src/data/matrix/basic.lean

```diff
@@ -716,6 +822,18 @@ by { ext, rw [←diagonal_one, mul_vec_diagonal, one_mul] }
 @[simp] lemma vec_mul_one [decidable_eq m] (v : m → α) : vec_mul v 1 = v :=
 by { ext, rw [←diagonal_one, vec_mul_diagonal, mul_one] }
 
+@[simp] lemma mul_vector_one (A : matrix m n α) : mul_vec A vec_one = λ i, ∑ j, A i j :=
```

Comment:
This lemma should be called `mul_vec_one`, and the lemma currently of that name renamed to `one_mul_vec`